ui/widgets/nuxeo-user-tag.js

```diff
@@ -169,6 +186,18 @@ import './nuxeo-tooltip.js';
     _preventPropagation(e) {
       e.stopPropagation();
     }
+
+    _computeLabel(user) {
+      return this._name(user);
```

Comment:
Why do we need this indirection?

Reply:
we dropped this use case for nuxeo-user-tag, since it includes a link (which provides accessibility by default in this context)